RobotCrawler/src/main/resources/robot-server.yml

```diff
@@ -0,0 +1,7 @@
+# Spring Properties
+spring:
+  application:
+     name: robot-app  # Service registers under this name
```

Comment:
Robot-server service registers in system as robot-server but here as robot-app? Talk about CONSISTENCY. :cop: